static/app/views/onboarding/onboardingController.tsx

```diff
@@ -1,17 +1,19 @@
+import {useEffect} from 'react';
+
+import {logExperiment} from 'sentry/utils/analytics';
 import withOrganization from 'sentry/utils/withOrganization';
 
 import TargetedOnboarding from './targetedOnboarding/onboarding';
 
 type Props = Omit<React.ComponentPropsWithoutRef<typeof TargetedOnboarding>, 'projects'>;
 
 function OnboardingController({...rest}: Props) {
-  // TODO: uncomment
-  // useEffect(() => {
-  //   logExperiment({
-  //     key: 'TargetedOnboardingMobileRedirectExperiment',
-  //     organization: rest.organization,
-  //   });
-  // }, [rest.organization]);
+  useEffect(() => {
```

Comment:
@Neo-Zhixing we should only log the experiment if it's a mobile user, right?

Reply:
Ah yeah my bad